### PR TITLE
block all unsafe ports

### PR DESCRIPTION
**Pull request by kasperk81**
extracted from three browsers' sources: https://github.com/dotnet/runtime/pull/76236#issuecomment-1262346000.

fix: #5379

### CODE REVIEW

File: src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Macros/Config/GeneratePortNumberConfig.cs [hunk 1]
```diff
@@ -12,15 +12,88 @@ internal class GeneratePortNumberConfig : IMacroConfig
...
+            554, // rtsp
+            556, // remotefs
+            563, // NNTP+SSL
+            587, // ESMTP
+            601, // syslog-conn
+            636, // LDAP+SSL
+            989, // ftps-data
+            990, // ftps
+            993, // IMAP+SSL
+            995, // POP3+SSL
```
Comment (vlada-shubina) [inline src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Macros/Config/GeneratePortNumberConfig.cs:77]: template engine blocks using of those already in the logic here:
https://github.com/dotnet/templating/blob/450bd360a9a6bb9204d16af3d8cce1d99d03e20c/src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Macros/GeneratePortNumberMacro.cs#L33-L83

The allowed port range is: 1024 - 65535

**Review by vlada-shubina — COMMENTED** [review]
In general, I'm ok with the change, not sure if we need to specify the ports below 1024 though. 

It would be good to add the content of the https://github.com/dotnet/runtime/pull/76236#issuecomment-1262346000 to code as the remark to know the source of information.

**Comment by JanKrivanek** [comment]
> In general, I'm ok with the change, not sure if we need to specify the ports below 1024 though.
> 
> It would be good to add the content of the [dotnet/runtime#76236 (comment)](https://github.com/dotnet/runtime/pull/76236#issuecomment-1262346000) to code as the remark to know the source of information.

The lower port bound is guarded to be >= 1024

https://github.com/dotnet/templating/blob/f9dc2bbe251d9814f2609b95601b650001460ece/src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Macros/GeneratePortNumberMacro.cs#L50-L53

**Review by vlada-shubina — APPROVED** [review]
Thank you

**Comment by JanKrivanek** [comment]
Thank you @kasperk81!

**Pull request merged**